miner/scroll_worker.go

```diff
@@ -796,7 +796,10 @@ func (w *worker) processTxns(txs types.OrderedTransactionSet) (bool, error) {
 			}
 
 			if tx.IsL1MessageTx() {
-				txs.Shift()
```

Comment:
if remove this, will `ErrUnexpectedL1MessageIndex` L1 message stop the L1 message to insert ?

Reply:
I think tx.Shift() shouldn't be removed.

Reply:
Calling `Shift` in case of an error means that we skipped the transaction and continue with the next one, which we should not do.

Instead, we'll be stuck at the erroring transaction, and we must fix that error before we continue processing L1 messages.

Reply:
understand. But I'm curious why it can be skipped to next before, but now we need to fix it immediately